manifests/base/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -112,7 +112,7 @@ spec:
         - --skip-provider-button=true
         - --openshift-ca=/etc/pki/tls/cert.pem
         - --openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-        image: quay.io/stolostron/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
+        image: quay.io/open-cluster-management/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
```

Comment:
You can use this tag:
https://github.com/stolostron/multicluster-observability-operator/blob/6f8f1a022262cbca02a22d0335c25641f08a6f68/operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml#L115

Reply:
thanks.